src/components/UpdatePage.js

```diff
@@ -0,0 +1,154 @@
+// react libraries
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import completeUserRegistration from '../action/completeUserRegistration';
+import decodedToken from '../helpers/decodeToken';
+
+// Moduler importations
+import Logo from './Logo';
+import '../styles/_form.scss';
+import '../styles/_update.scss';
+/**
+ * @class CreateArticle
+ * @extends {Component}
+ * @param {object} event
+ */
+class UpdatePage extends Component {
+  constructor(props) {
+    super(props);
+
+    const { token } = this.props.match.params.token;
+    const userData = decodedToken(token);
+
+    this.state = {
+      email: userData.email,
+      password: '',
+      fullname: '',
+      username: '',
+      confirmPassword: '',
+      submitted: false,
+      token
+    };
+
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+
+  handleChange(e) {
```

Comment:
#### Critical
This is the `event` object so write the full variable name instaed of `e`